contracts/gasToken/GasTokenForwarder.sol

```diff
@@ -0,0 +1,122 @@
+pragma solidity ^0.5.13;
+import "./IGST2.sol";
+import "../libs/SafeMath.sol";
+import "../libs/Ownable.sol";
+
+contract GasTokenForwarder is Ownable {
+    using SafeMath for uint256;
+    
+    IGST2 private _gasToken;
+    event Efficiency(uint256 indexed efficiency, uint256 indexed tokens);
+    
+    constructor(address owner, address gasToken2) public Ownable(owner){
+        _gasToken = IGST2(gasToken2);
+    }
+    
+    function forward(address payable destination, bytes memory data,uint256 mintPrice,uint256 gasLimit) public { 
```

Comment:
Add onlyRelayers modifier.
Deploy RelayerManager contract also and initialize its instance in constructor using its address.